CODING_STYLE.md

```diff
@@ -0,0 +1,39 @@
+# Coding Style
+
+## General
+
+When contributing to the project, and if otherwise not mentioned in this document, our coding conventions
```

Comment:
I've proposed discussion about on-the-line comments into How we work not scheduled discussion. I'd like to allow it (and therefore mention it here). I found them practical and they appear a lot in test cases by-design (Noncompliant, Secondary).

Should we have a quick team discussion first? Or are we going to update that document later (little overhead)?